app/screens/channel_info/title/public_private/public_private.tsx

```diff
@@ -25,11 +34,62 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     },
 }));
 
+const style = StyleSheet.create({
+    bottomsheet: {
+        flex: 1,
+    },
+});
+
 const PublicPrivate = ({displayName, purpose}: Props) => {
+    const intl = useIntl();
     const theme = useTheme();
+    const managedConfig = useManagedConfig<ManagedConfig>();
+    const {bottom} = useSafeAreaInsets();
+
     const styles = getStyleSheet(theme);
     const publicPrivateTestId = 'channel_info.title.public_private';
 
+    const onCopy = useCallback(() => {
+        Clipboard.setString(purpose!);
+        dismissBottomSheet().then(() => {
```

Comment:
We recently have seen that when we copy something into the clipboard, later Android version add an OS message, so this message becomes redundant.

Here is the PR that solves it in a different place. Feel free to apply this here, or take a look on how can it be done more centralized (maybe in `showSnackBar`? 0/5)

https://github.com/mattermost/mattermost-mobile/pull/7578/files

Reply:
Done 👍🏻